pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+testpaths = unit_test test
+norecursedirs = spamtest
```

Comment:
Can anyone explain me what these do?

Reply:
OK, nvm, I understand. It's because sometimes we want to run tests inside of cibw during the tests. Testception.

Reply:
`testpaths = unit_test test` - in which directories there are test
`norecursedirs = spamtest` - not try to discovery test in this directories (test 02)